src/aws/gateway.ts

```diff
@@ -241,10 +238,10 @@ export class AWSGateway implements bs.Gateway {
   }
 
   async subscribe(url: URI, callback: (meta: Uint8Array) => void): Promise<bs.UnsubscribeResult> {
-    url = url.build().setParam("key", "main").URI();
+    url = url.build().setParam("key", "main").defParam("interval", "100").URI();
 
     let lastData: Uint8Array | undefined = undefined;
-    let interval = 100;
+    let interval = parseInt(url.getParam("interval") || "100", 10);
```

Comment:
thanks

Reply:
`interval = Math.min(interval * 2, 3000);` is maybe the more important value, maybe want both to be config

Reply:
i done it in a different place.